torch/csrc/jit/codegen/cuda/python_frontend/README.md

```diff
@@ -0,0 +1,119 @@
+# nvFuser Python Frontend
+
+This frontend allows for a user to describe the set of operations for nvFuser to fuse via 1 or more kernels.  This frontend is intended to be an integration point with Pytorch or standalone applications.
+
+# Usage
+
+## Example
+
+```
```

Comment:
nit-picking: I vaguely remember GitHub can have syntax highlight in code blocks
```suggestion
```python
```

Reply:
Thanks, I didn't know that!